pkg/piecestore/psserver/config.go

```diff
@@ -26,10 +26,11 @@ var (
 
 // Config contains everything necessary for a server
 type Config struct {
-	Path                   string        `help:"path to store data in" default:"$CONFDIR"`
-	AllocatedDiskSpace     int64         `help:"total allocated disk space, default(1GB)" default:"1073741824"`
-	AllocatedBandwidth     int64         `help:"total allocated bandwidth, default(100GB)" default:"107374182400"`
-	KBucketRefreshInterval time.Duration `help:"how frequently checker should audit segments" default:"3600s"`
+	Path                         string        `help:"path to store data in" default:"$CONFDIR"`
+	AllocatedDiskSpace           int64         `help:"total allocated disk space in bytes, default(1GB)" default:"1073741824"`
+	AllocatedBandwidth           int64         `help:"total allocated bandwidth in bytes, default(100GB)" default:"107374182400"`
+	KBucketRefreshInterval       time.Duration `help:"how frequently Kademlia bucket should be refreshed with node stats" default:"3600s"`
```

Comment:
`3600s` is 1 hour, right? How about changing the default to `1h0m0s` so it's clearer?

Reply:
done